### ISSUE TITLE
Unable to install package in Laravel v5.2 and in Laravel v5.5

### ISSUE THREAD

**Issue by alanvallejodev** [spiderwebtr/isauth#9]
Hi,

I've tried to install the package in Laravel version v5.2 and v5.5 and in both, everytime the method boot is called in the "isAuthServiceProvider", the following error is returned:

```
In Facade.php line 221:
  Call to undefined method Illuminate\View\Compilers\BladeCompiler::include()
```

Have you had any issues with this before?

Thanks for your time!

**Comment by emredipi** [comment]
Hello @alanvallejodev, 
Thanks for the feedback. When I created this package I was working new versions. I created Laravel 5.2 and 5.5 projects to test but unfortunately the package uses new methods from serviceprovider and blade.
Can you please upgrade your project at least laravel 5.6?

**Comment by alanvallejodev** [comment]
I was making the same test at the moment :), obviously the "Blade::include()" method is only available as from laravel v5.6.

I will try to upgrade to v5.6.

Thanks for your quick response!

**Comment by emredipi** [comment]
I prepared a demo project. 😊
https://isauth.herokuapp.com/
https://github.com/spiderwebtr/isAuthDemo
